<launch>

  <node name="trajectory_handler" pkg="panda_ros" type="training_trajectory_generator.py"
        output="screen"/>

  <node name="$(anon sample_buttons)"
        pkg="jsk_rqt_plugins" type="rqt_service_buttons" output="screen" clear_params="true">
    <param name="~layout_yaml_file" value="package://panda_ros/resources/trajectory_recorder_layout.yaml" />
  </node>
  
</launch>
